### PR TITLE
css on register done

### PR DESCRIPTION
**Pull request by AllanCaplain**

### CODE REVIEW

File: src/pages/Register.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.box1{
+    display: grid;
+    place-items: center;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 10%;
```
Comment (rocambille) [inline src/pages/Register.css:7]: you should use rem for margins, and no more than 4rem: if you need more, you're facing a layout problem, not spacing. try flexbox/grid on the container instead

File: src/pages/Register.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+.child{
+    border: 1px solid black;
+    display: grid;
+    place-items: center;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 3vh;
```
Comment (rocambille) [inline src/pages/Register.css:16]: you should use rem for padding, and no more than 4rem: if you need more, you're facing a layout problem, not spacing. try flexbox/grid on the container instead

File: src/pages/Register.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    padding: 0.25rem;
+}
+label{
+    font-size:2rem;
+}
+.button1{
+    font-size: 1.50rem;
+    background-color: black;
+    border-radius: 10px;
+    margin-top: 5vh;
```
Comment (rocambille) [inline src/pages/Register.css:30]: you should use rem for margins, and no more than 4rem: if you need more, you're facing a layout problem, not spacing. try flexbox/grid on the container instead

File: src/pages/Register.css (hunk 1 above)
Comment (AllanCaplain) [reply src/pages/Register.css:7]: for margin here, I used it for verticaly center my div In the page, so I don't know how replace it

Comment (rocambille) [reply src/pages/Register.css:7]: try flexbox/grid :)

Comment (AllanCaplain) [reply src/pages/Register.css]: anyway, we don't need margin anymore here, so conflicts are resolved :)

**Pull request merged**